### PR TITLE
US 1583733 Add missing language IDs - 13

### PR DESCRIPTION
**Pull request by v-thpra**
US 1583733 - Fix CATS P1 issues by adding missing language IDs to code blocks.
- Used "console" for EdmGen.exe command examples.
- Used "csharp" for query command tree examples.
- Used "csharp" for Entity SQL syntax examples.

Contributes to #2192

### CODE REVIEW

**Comment by v-thpra** [comment]
@mairaw Could you please review and comment on the last 10 changes especially. They are syntax examples for Entity SQL. There appear to be many more in the list, so I want to get it correct now.  Thanks!

**Comment by mairaw** [comment]
@divega @stevestein can you guys help with this one? Should the proper identifier for those entity SQL expressions be sql?

**Review by BillWagner — APPROVED** [review]
I'll approve this and :shipit: now.

I checked other samples in dotnet/docs, and `sql` is the tag used for other sql statements.

/cc @mairaw

File: docs/framework/data/adonet/ef/connection-strings.md [hunk 1]
```diff
@@ -51,39 +49,39 @@ The following options are available for defining the location of an embedded res
...
+```csharp
 Metadata=res://*/
 ```
 
 The following example loads the model.csdl file from the AdventureWorks assembly, and loads the model.ssdl and model.msl files from the default directory of the running application.
 
-```
+```csharp
 Metadata=res://AdventureWorks, 1.0.0.0, neutral, a14f3033def15840/model.csdl|model.ssdl|model.msl
 ```
```
Comment (Youssef1313) [inline docs/framework/data/adonet/ef/connection-strings.md]: I'm not convinced by these being csharp.
/cc @BillWagner @mairaw

**Pull request merged**